Automattic-Tracks-iOS/Model/UploadError.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public enum UploadError: Error, LocalizedError {
+    case httpError(String)
+    case fileMissing
+
+    public var errorDescription: String?{
+        switch self {
+            case .httpError(let message): return message
+            case .fileMissing: return "File not found"
```

Comment:
Hmmm – good point. It _shouldn't_ need it, but we can't guarantee that, so I've enabled translation for it just in case in ac7532802aef539c248490c9f10a142817194b70.